drivers/video/ov7725.c

```diff
@@ -0,0 +1,651 @@
+#define OV7725_LC_CTR     0x46U
+#define OV7725_LC_XC      0x47U
+#define OV7725_LC_YC      0x48U
+#define OV7725_LC_COEF    0x49U
+#define OV7725_LC_RADI    0x4AU
+#define OV7725_LC_COEFB   0x4BU
+#define OV7725_LC_COEFR   0x4CU
+#define OV7725_FIXGAIN    0x4DU
+#define OV7725_AREF1      0x4FU
+#define OV7725_AREF6      0x54U
+#define OV7725_UFIX       0x60U
+#define OV7725_VFIX       0x61U
+#define OV7725_AWBB_BLK   0x62U
+#define OV7725_AWB_CTRL0  0x63U
+#define OV7725_DSP_CTRL1  0x64U
+#define OV7725_DSP_CTRL2  0x65U
+#define OV7725_DSP_CTRL3  0x66U
+#define OV7725_DSP_CTRL4  0x67U
+#define OV7725_AWB_BIAS   0x68U
+#define OV7725_AWB_CTRL1  0x69U
+#define OV7725_AWB_CTRL2  0x6AU
+#define OV7725_AWB_CTRL3  0x6BU
+#define OV7725_AWB_CTRL4  0x6CU
+#define OV7725_AWB_CTRL5  0x6DU
+#define OV7725_AWB_CTRL6  0x6EU
+#define OV7725_AWB_CTRL7  0x6FU
+#define OV7725_AWB_CTRL8  0x70U
+#define OV7725_AWB_CTRL9  0x71U
+#define OV7725_AWB_CTRL10 0x72U
+#define OV7725_AWB_CTRL11 0x73U
+#define OV7725_AWB_CTRL12 0x74U
+#define OV7725_AWB_CTRL13 0x75U
+#define OV7725_AWB_CTRL14 0x76U
+#define OV7725_AWB_CTRL15 0x77U
+#define OV7725_AWB_CTRL16 0x78U
+#define OV7725_AWB_CTRL17 0x79U
+#define OV7725_AWB_CTRL18 0x7AU
+#define OV7725_AWB_CTRL19 0x7BU
+#define OV7725_AWB_CTRL20 0x7CU
+#define OV7725_AWB_CTRL21 0x7DU
+#define OV7725_GAM1       0x7EU
+#define OV7725_GAM2       0x7FU
+#define OV7725_GAM3       0x80U
+#define OV7725_GAM4       0x81U
+#define OV7725_GAM5       0x82U
+#define OV7725_GAM6       0x83U
+#define OV7725_GAM7       0x84U
+#define OV7725_GAM8       0x85U
+#define OV7725_GAM9       0x86U
+#define OV7725_GAM10      0x87U
+#define OV7725_GAM11      0x88U
+#define OV7725_GAM12      0x89U
+#define OV7725_GAM13      0x8AU
+#define OV7725_GAM14      0x8BU
+#define OV7725_GAM15      0x8CU
+#define OV7725_SLOP       0x8DU
+#define OV7725_DNSTH      0x8EU
+#define OV7725_EDGE0      0x8FU
+#define OV7725_EDGE1      0x90U
+#define OV7725_DNSOFF     0x91U
+#define OV7725_EDGE2      0x92U
+#define OV7725_EDGE3      0x93U
+#define OV7725_MTX1       0x94U
+#define OV7725_MTX2       0x95U
+#define OV7725_MTX3       0x96U
+#define OV7725_MTX4       0x97U
+#define OV7725_MTX5       0x98U
+#define OV7725_MTX6       0x99U
+#define OV7725_MTX_CTRL   0x9AU
+#define OV7725_BRIGHT     0x9BU
+#define OV7725_CNST       0x9CU
+#define OV7725_UVADJ0     0x9EU
+#define OV7725_UVADJ1     0x9FU
+#define OV7725_SCAL0      0xA0U
+#define OV7725_SCAL1      0xA1U
+#define OV7725_SCAL2      0xA2U
+#define OV7725_SDE        0xA6U
+#define OV7725_USAT       0xA7U
+#define OV7725_VSAT       0xA8U
+#define OV7725_HUECOS     0xA9U
+#define OV7725_HUESIN     0xAAU
+#define OV7725_SIGN       0xABU
+#define OV7725_DSPAUTO    0xACU
+
+#define OV7725_COM10_VSYNC_NEG_MASK    (1U << 1U)
+#define OV7725_COM10_HREF_REVERSE_MASK (1U << 3U)
+#define OV7725_COM10_PCLK_REVERSE_MASK (1U << 4U)
+#define OV7725_COM10_PCLK_OUT_MASK     (1U << 5U)
+#define OV7725_COM10_DATA_NEG_MASK     (1U << 7U)
```

Comment:
Recommend using `BIT()` from `include/sys/util.h`

Reply:
Thanks for review, Fixed